specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
And if it doesn't contain a "BallotType", how do we figure out what contests to list on the auditor dashboard for that ballot when it gets randomly selected?